devtools-ci-index.yaml

```diff
@@ -1641,6 +1641,80 @@
                 variable: EE_TEST_KC_TOKEN
     <<: *job_template_end_to_end_tests_defaults
 
+- job-template:
+    name: 'devtools-test-e2e-{test_url}-{test_suite}-{cluster}-{feature_level}'
+    defaults: global
+    node: devtools
+    timeout: 60m
+    zabbix_enabled: ''
+    properties:
+        - github:
+            url: https://github.com/fabric8io/fabric8-test
+    scm:
+        - git:
+            url: https://github.com/fabric8io/fabric8-test.git
+            shallow_clone: true
+            branches:
+                - master
+    triggers:
+        - timed: '{ee_test_start_time}'
+    wrappers:
+        - credentials-binding:
+            - username-password-separated:
+                credential-id: "{osio_creds}"
+                username: OSIO_USERNAME
+                password: OSIO_PASSWORD
+    builders:
+        - shell: |
+            set +e
+            set +x
+            export OSIO_CLUSTER="{cluster}"
+            export OSIO_URL="https://{test_url}"
+            export TEST_SUITE="{test_suite}"
+            export FEATURE_LEVEL="{feature_level}"
+            export ZABBIX_ENABLED="{zabbix_enabled}"
+            env > jenkins-env
+            cat $creds_config_file >> jenkins-env
+            cp ~/artifacts.key .
+            export CICO_API_KEY=$(cat ~/duffy.key )
+            # get node
+            n=1
+            while true
+            do
+                cico_output=$(cico node get -f value -c ip_address -c comment)
+                if [ $? -eq 0 ]; then
+                    read CICO_hostname CICO_ssid <<< $cico_output
+                    if  [ ! -z "$CICO_hostname" ]; then
+                        # we got hostname from cico
+                        break
+                    fi
+                    echo "'cico node get' succeed, but can't get hostname from output"
+                fi
+                if [ $n -gt 5 ]; then
+                    # give up after 5 tries
+                    echo "giving up on 'cico node get'"
+                    exit 1
+                fi
+                echo "'cico node get' failed, trying again in 60s ($n/5)"
+                n=$[$n+1]
+                sleep 60
+            done
+            echo 'Using Host' $CICO_hostname
+            set -x
+            sshopts="-t -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -l root"
+            ssh_cmd="ssh $sshopts $CICO_hostname"
+            $ssh_cmd yum -y install rsync
+            rsync -e "ssh $sshopts" -Ha $(pwd)/ $CICO_hostname:payload
+            /usr/bin/timeout {timeout} $ssh_cmd -t "cd payload/ee_tests && /bin/bash cico_run_e2e_tests.sh"
+            rtn_code=$?
+            if [[ $rtn_code -eq 124 ]]; then
+               echo "BUILD TIMEOUT";
+            else
+               rsync -delete -PHva -e "ssh $sshopts" $CICO_hostname:payload/ $(pwd) || rtn_code=$?
```

Comment:
I know this is in other parts of the index file as well, but what purpose does it serve?

Reply:
Frankly, I don't understand this command so I simply left it as it was in old template.

Reply:
Clears the payload directory after the job script has been run successfully without a timeout, so that the node can later be used for a different payload? 

@jmelis doesn't CICO node done do that, by default?

Reply:
I do not see any active job is doing that so I suppose it is done automatically

Reply:
do you want me to remove that line?

Reply:
No, it's fine, I think this is just leftover cruft. I will try to clean these things in one go.